babi/screen.py

```diff
@@ -225,11 +225,15 @@ def _get_string(self, wch: str) -> str:
     def _get_char(self) -> Key:
         if self._buffered_input is not None:
             wch, self._buffered_input = self._buffered_input, None
-        else:
+        while True:
             try:
                 wch = self.stdscr.get_wch()
             except curses.error:  # pragma: no cover (macos bug?)
-                wch = self.stdscr.get_wch()
+                pass
+            else:
+                break
+        else:
+            wch = self.stdscr.get_wch()
```

Comment:
this `else` block is impossible to run as `while True` [can never exit gracefully](https://www.youtube.com/watch?v=8P7lXLXR_3c&list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY&index=13)